dist/jquery.bxslider.js

```diff
@@ -1103,7 +1103,7 @@
     var onTouchStart = function(e) {
       // watch only for left mouse, touch contact and pen contact
       // touchstart event object doesn`t have button property
-      if (e.type !== 'touchstart' && e.button !== 0) {
+      if ((e.type !== 'touchstart' ||  e.type !== 'pointerdown') && e.button !== 0) {
```

Comment:
This change doesn't make any sense.
Your proposed `(e.type !== 'touchstart' ||  e.type !== 'pointerdown')` could also be written as `!(e.type === 'touchstart' && e.type === 'pointerdown')`. As you probably can see now, this would _always_ be true,...